wiki/publications.md

```diff
@@ -32,16 +32,46 @@ article in *Computational Materials Science* entitled ["Phase field
 benchmark problems for dendritic growth and linear
 elasticity"](http://dx.doi.org/10.1016/j.commatsci.2016.09.022){:target="_blank"}.
 
+No preprint version available.
+
 <iframe width="900"
         height="700"
         src="https://www.sciencedirect.com/science/article/pii/S092702561830168X?via%3Dihub"
         frameborder="0"
         allowfullscreen>
 </iframe>
 
+<br>
+<br>
+
+## Publication about PFHub
+
+[Daniel Wheeler][dw], [Trevor Keller][tk], [Steve DeWitt][sd], [Andrea
+Jokisaari][aj], [Daniel Schwen][ds], [Jon Guyer][jg], [Larry
+Aagesen][la], [Olle Heinonen][oh], [Michael Tonks][mt], [Peter
+Voorhees][pv] and [James Warren][jw] published a peer-reviewed journal
+article in *Journal of open research software* entitled ["PFHub: The
+Phase-Field Community
+Hub"](http://dx.doi.org/10.1016/j.commatsci.2016.09.022){:target="_blank"}.
+
+<iframe width="900"
+        height="700"
+        src="https://drive.google.com/file/d/1iJvZzaxAQjNYpDkl20Uyb33HbDKfys8x/preview"
```

Comment:
As *JORS* is open-access, the DOI link (https://doi.org/10.5334/jors.276) would be preferable to this bootleg copy.

Reply:
Doesn't work unfortunately. Can't be embedded and is not a PDF view either. JORS also doesn't have a PDF viewer so if I want to embed the view then I need another place to host the PDF. Also, no such thing as a bootleg copy with [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).